config/locales/user_mailer/en.yml

```diff
@@ -286,6 +286,11 @@ en:
         sign in with this email address, you can ignore this message.
       link_text: Go to %{app_name}
       reset_password_html: If you can’t remember your password, go to %{app_name_html} to reset it.
+    suspended_create_account:
+      message: There was an issue creating your %{app_name_html} account with this
```

Comment:
looks like we are not passing in `app_name_html` in the template.

So this is part of the new lint that I added in #8713.

Either:
- we need to rename this to `message_html` because we expect HTML inside of the `%{app_name_html}`
- we need to rename `%{app_name_html} => %{app_name}` because there is not going to be HTML there